bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/thing/channelTypes.xml

```diff
@@ -134,12 +142,20 @@
 		<item-type>Switch</item-type>
 		<label>@text/channel-type.mielecloud.program_start.label</label>
 		<description>@text/channel-type.mielecloud.program_start.description</description>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Mode</tag>
+		</tags>
 	</channel-type>
 
 	<channel-type id="program_stop">
 		<item-type>Switch</item-type>
 		<label>@text/channel-type.mielecloud.program_stop.label</label>
 		<description>@text/channel-type.mielecloud.program_stop.description</description>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Mode</tag>
```

Comment:
This is equivalent to https://github.com/openhab/openhab-addons/pull/18585#discussion_r2055611276.

Reply:
Ok. Deleted.